site.maintenance.aws-remove-all.yml

```diff
@@ -53,6 +52,38 @@
             wait: yes
       when: remove_ec2
 
+    - name: Remove RDS
+      block:
+        - name: Get list of RDS instances
+          rds_instance_facts:
+            region: "{{ aws_region }}"
+          register: _rds_instances_info
+        - name: Set list of rds instances
+          set_fact:
+            _aws_rds_instances: "{{ _rds_instances_info.instances }}"
+        - name: Filter rds instance list by tags
+          set_fact:
+            _aws_rds_instances: "{{ _aws_rds_instances | json_query(rds_instances_tag_filter_query) | default([]) }}"
+          vars:
+            rds_instances_tag_filter_query: "[?{% for k, v in aws_rds_facts_mysql_tags.items() -%}tags.{{ k }} == '{{ v }}'{% if not loop.last %} && {% endif %}{% endfor %}]"
+        - name: Warn when more than one instance has been found
+          debug:
+            msg: |
+              Warning! More than one matching rds instance found, using first one.
+              Found: {{ _aws_rds_instances | map(attribute='db_instance_identifier') | join(', ') }}
+          when: _aws_rds_instances | length > 1
+        - name: Set facts about project's rds instance
+          set_fact:
+            aws_rds_instance_id: "{{ (_aws_rds_instances | first).db_instance_identifier }}"
+          when: _aws_rds_instances | length > 0
+        - name: Terminate RDS instance
+          when: _aws_rds_instance_id == aws_rds_instance_name
+          rds:
+            command: delete
+            instance_name: "{{ aws_rds_instance_name }}"
+            # snapshot: "{{ aws_rds_instance_name }}-snapshot"
```

Comment:
I suggest having this as variable, and when the variable is empty than omit the parameter (|default(omit))

something like `aws_rds_final_snapshoot` with a default of "{{ aws_rds_instance_name }}-final" an possibility to set null (via vars or extra parameters)